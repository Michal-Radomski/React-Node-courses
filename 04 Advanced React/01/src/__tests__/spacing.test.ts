import spacing from "../spacing";

test("snapshot of spacing", (): void => {
  expect(spacing).toMatchSnapshot();
});
