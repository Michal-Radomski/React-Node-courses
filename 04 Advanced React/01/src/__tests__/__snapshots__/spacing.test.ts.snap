// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot of spacing 1`] = `
{
  "lg": "lg",
  "md": "md",
  "none": "none",
  "sm": "sm",
  "xl": "xl",
  "xs": "xs",
  "xxl": "xxl",
  "xxs": "xxs",
  "xxxl": "xxxl",
  "xxxs": "xxxs",
}
`;
